specsim/config.py

```diff
@@ -370,7 +374,7 @@ def load_table(self, parent, column_names, interpolate=True, as_dict=False):
             # Look for parent.table.path first.
             paths.append(os.path.join(self.abs_base_path, node.path))
         except AttributeError:
-            path_keys = node.paths.keys()
+            path_keys = list(node.paths.keys())
```

Comment:
Not really a Python 3 problem, but I see this so often.  `for key in node.paths` is just so much easier.